common/models/role.js

```diff
@@ -240,6 +241,15 @@ module.exports = function(Role) {
     Role.isAuthenticated(context, callback);
   });
 
+  Role.registerResolver(Role.EVERYONE_PROMISE, function(role, context) {
```

Comment:
@bajtos I registered a new resolver since we don't have any existed resolver that returns promise.
They all use this signature: [function(role, context, callback)](https://github.com/strongloop/loopback/blob/master/common/models/role.js#L124)

But my test need to verify signature that ignore callback but return a promise.